src/clj/doit/todo/db.clj

```diff
@@ -0,0 +1,11 @@
+(ns doit.todo.db
+  (:require [clojure.java.jdbc :as jdbc]
+            [doit.config :as config]))
+
+(defn add-todo!
+  [values]
+  (jdbc/insert! config/db :todo values))
+
+(defn list-todos
+  []
+  (jdbc/query config/db ["SELECT * FROM todo"]))
```

Comment:
It's possible to `keywordize` the output here by passing a `{:row-fn some-fn}` to the `query` fn.

Reply:
Output from `jdbc/insert!` or `jdb/query` is `keywordize` by default. See [change log for jdbc 0.7.3](https://github.com/clojure/java.jdbc/blob/master/CHANGES.md)